polars/polars-lazy/src/physical_plan/expressions/literal.rs

```diff
@@ -58,21 +62,18 @@ impl PhysicalExpr for LiteralExpr {
                     ca.into_inner().into_series()
                 }
                 DataType::UInt32 => {
-                    if *low >= 0 || *high <= u32::MAX as i64 {
```

Comment:
Looks like this was backwards

Reply:
Yeap, shows the benefit of reasoning with `ensure`. :)